back-end/controllers/snackController.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const snacks = express.Router();
+const confirmHealth = require("../confirmHealth.js");
+const checkName = require('../validations/checkName.js');
+
+//QUERIES//
+const {
+  getAllSnacks,
+  getSnack,
+  createSnack,
+  updateSnack,
+  deleteSnack,
+} = require('../queries/snacks');
+
+snacks.get("/", async (req, res) => { 
+    const allSnacks = await getAllSnacks();
+    if(allSnacks[0]){
+        res.status(200).json({payload: allSnacks, success: true});
+    }else{
```

Comment:
we should also get in the habit of spacing out our `if` and `else`

`if (condition) {`
` } else { ` 
its minor but a good habit to get into